tools/ffi_build.py

```diff
@@ -0,0 +1,9 @@
+# https://cffi.readthedocs.io/en/latest/overview.html#main-mode-of-usage
+
+from cffi import FFI
+
+ffibuilder = FFI()
+ffibuilder.cdef("signed int lsl_protocol_version();")
+ffibuilder.set_source("mne_lsl._ffi_liblsl", '#include "lsl_c.h"', libraries=["lsl"])
+if __name__ == "__main__":
+    ffibuilder.compile(verbose=True)
```

Comment:
... no, not yet :)

So far, `cffi` is used *only* as a convenient way to force delocation tools to bundle liblsl and create `abi3` wheels. And in the wheel tests you can see I install `cffi` and make sure it can access the library, that's it. Everything else still runs through the old `ctypes` code, just knowing that our dynamic library finding code will look in the right place.

In the long run, yes in principle `cffi` could probably be used to replace all the `ctypes` type setting that is being done. But that's a larger, separable undertaking.

Reply:
Oh, so `cffi` is needed because if it was absent, `auditwheel` / `delocate-wheel` / `delvewheel` would not bundle `liblsl` since `mne-lsl` doesn't use it anywhere. Which is not the case of `antio` since it has a direct call `from . import pyeep` in `src/antio/libeep/__init__.py`. Correct?

Reply:
Yes exactly!